src/core/public/chrome/ui/header/variables.scss

```diff
@@ -0,0 +1 @@
+$ouiSideNavBackgroundColorTemp: lightOrDarkTheme(#ebe4df, #001c28); /* stylelint-disable-line */ 
```

Comment:
should this go with theme change? when i switch to theme v7, the page looks like this

![image](https://github.com/user-attachments/assets/ee34b8bd-08ce-4ec1-a6dc-cbe6b687941c)

Reply:
It is. a intermedia state, will change to a variable in OUI theme after https://github.com/opensearch-project/oui/pull/1430/files#diff-67cb04d06a675c67602ff65a876373033b16a759964be1b850bbfb4ca50bad6aR13 get merged.